core/state/snapshot/snapshot.go

```diff
@@ -634,32 +620,26 @@ func diffToDisk(bottom *diffLayer) *diskLayer {
 	// Update the snapshot block marker and write any remainder data
 	rawdb.WriteSnapshotRoot(batch, bottom.root)
 
-	// Write out the generator progress marker and report
-	journalProgress(batch, base.genMarker, stats)
```

Comment:
It's unnecessary to journal the progress, reasons:

- If the background generation was active, the latest progress must be persisted before terminating it
- If the background generation was not active, no progress has been made, no need to persist marker